selfdrive/car/hyundai/values.py

```diff
@@ -1496,8 +1496,10 @@ def match_fw_to_car_fuzzy(live_fw_versions) -> Set[str]:
       b'\xf1\x8758520-K4010\xf1\x00OS IEB \x02 101 \x11\x13 58520-K4010',
       b'\xf1\x8758520-K4010\xf1\x00OS IEB \x04 101 \x11\x13 58520-K4010',
       b'\xf1\x8758520-K4010\xf1\x00OS IEB \x03 101 \x11\x13 58520-K4010',
+      b'\xf1\x8758520-K4010\xf1\x00OS IEB \x03 102"\x05\x16 58520-K4010',
```

Comment:
```suggestion
```

Reply:
This part is different? \x03 102"\x05\x16